spec/controllers/bets_controller_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe BetsController, type: :controller do
+  describe 'GET #index' do
+    subject { get :index }
+    it { is_expected.to be_ok }
+  end
+
+  describe 'GET #show' do
+    subject { get :show, params: { id: bet.id } }
+    let(:user) do
+      User.create(username: 'Bob', email: 'bob@mail.com', password: '123')
+    end
+    let(:bet) do
+      Bet.create(bet: 'Bet text',
+                 expires_at: DateTime.tomorrow,
+                 creator_id: user.id)
+    end
+    it { is_expected.to be_ok }
+  end
+
+  describe 'GET #new' do
+    subject { get :new }
+    it { is_expected.to be_ok }
+  end
+
+  describe 'POST #create' do
+    subject(:post_create) { post :create, params: { bet: bet_params } }
```

Comment:
Yarek has told me in person the reason he made these changes and they make sense to me.